client/src/c_bind.cpp

```diff
@@ -729,15 +730,15 @@ C_GetKeyStringsFromCommand
 Finds binds from a command and returns it into a std::string .
 - If TRUE, second arg returns up to 2 keys. ("x OR y")
 */
-std::string C_GetKeyStringsFromCommand(char *cmd, bool bTwoEntries)
+std::string C_GetKeyStringsFromCommand(const char *cmd, bool bTwoEntries)
 {
 	int first = -1;
 	int second = -1;
 
 	C_GetKeysForCommand(cmd, &first, &second);
 
 	if (!first && !second)
-		return "<???>";
+		return "< ??? >";
```

Comment:
What happens if you use <\?\?\?> instead, to avoid sending spaces to the HUD ? (unbind your `ready` key, and join a warmup server to see the string in action)

Reply:
The compiler warns about ignored trigraphs, specifically ??>. I found no way to keep it as is so I added spaces on both sides. https://stackoverflow.com/questions/1234582/purpose-of-trigraph-sequences-in-c

Reply:
Actually I could just use \?.

Reply:
Long after the original commit, I modified it with \?\?\? to avoid that warning without pushing it, but if ??\? also works, then it's good to me.

Reply:
Commit added addressing this.